src/game/engine/team.h

```diff
@@ -106,11 +107,11 @@ class TeamClass : public AbstractClass
     int m_field_35;
     FormationType m_Formation;
     TCountDownTimerClass<FrameTimerClass> m_field_3A;
-    GamePtr<TriggerTypeClass> m_Trigger;
+    GamePtr<TriggerClass> m_Trigger;
     int m_Mission;
     TCountDownTimerClass<FrameTimerClass> m_field_4B;
     FootClass *m_field_54;
```

Comment:
Info for this suggests it's best to name this "m_Members" if this is a linked list.

Reply:
The team stuff can be sorted in a proper TeamClass focussed PR.